dist/tools/openocd/openocd.sh

```diff
@@ -334,6 +383,7 @@ do_debug() {
             ${OPENOCD_DBG_EXTRA_CMD} \
             -c 'targets' \
             ${OPENOCD_DBG_START_CMD} \
+            -c 'targets ${OPENOCD_CORE}' \
```

Comment:
Please document this variable as done for other variables in the script.

Reply:
ack